box.js

```diff
@@ -13,8 +13,8 @@ module.exports = function box (plain_text, feed_id, prev_msg_id, msg_key, recp_k
   const derive = DeriveSecret(feed_id, prev_msg_id)
 
   const read_key = derive(msg_key, [LABELS.read_key])
-    const header_key = derive(read_key, [LABELS.header_key])
-    const body_key   = derive(read_key, [LABELS.body_key])
```

Comment:
These indents were reminders to the developer about the heirarchy of these keys.
I'd like to leave it like this (or similar) to make the code easier to read.

I would also be fine with putting a comment in if that's better:

```js
// read_key
//   ├──> header_key
//   └──> body_key
```

Reply:
Yeah a comment would be much more clear. I didn't see it from the indentation sand it just looked like incorrect indentation